### ISSUE TITLE
Object serialized to integer

### ISSUE THREAD

**Issue by pts211** [Ladder99/fanuc-driver#60]
When running against a two path Fanuc CNC I'm getting the error shown below. This will occur if I enable either of the following collections, with the stack referencing either AXIS or SPINDLE accordingly.
- l99.driver.fanuc.collectors.AxisData, fanuc
- l99.driver.fanuc.collectors.SpindleData, fanuc

```text
2022/05/25 20:45:53.860|ERROR|[taki] Strategy sweep failed at segment SPINDLE. System.ArgumentException: Object serialized to Integer. JObject instance expected.
   at Newtonsoft.Json.Linq.JObject.FromObject(Object o, JsonSerializer jsonSerializer)
   at Newtonsoft.Json.Linq.JObject.FromObject(Object o)
   at System.Dynamic.UpdateDelegates.UpdateAndExecute2[T0,T1,TRet](CallSite site, T0 arg0, T1 arg1)
   at l99.driver.base.Veneer.onErrorAsync(Object input) in /src/base-driver/base/Veneer.cs:line 99
   at System.Dynamic.UpdateDelegates.UpdateAndExecuteVoid1[T0](CallSite site, T0 arg0)
   at l99.driver.fanuc.veneers.SpindleData.AnyAsync(Object input, Object[] additionalInputs) in /src/fanuc/veneers/SpindleData.cs:line 127
   at CallSite.Target(Closure , CallSite , Object )
   at l99.driver.base.Veneer.FirstAsync(Object input, Object[] additionalInputs) in /src/base-driver/base/Veneer.cs:line 118
   at CallSite.Target(Closure , CallSite , Object )
   at l99.driver.base.Veneer.PeelAsync(Object input, Object[] additionalInputs) in /src/base-driver/base/Veneer.cs:line 132
   at CallSite.Target(Closure , CallSite , Object )
   at l99.driver.base.Veneers.PeelAcrossAsync(Object split, String name, Object input, Object[] additionalInputs) in /src/base-driver/base/Veneers.cs:line 102
   at CallSite.Target(Closure , CallSite , Object )
   at l99.driver.base.Machine.PeelAcrossVeneerAsync(Object split, String name, Object input, Object[] additionalInputs) in /src/base-driver/base/Machine.cs:line 182
   at CallSite.Target(Closure , CallSite , Object )
   at System.Dynamic.UpdateDelegates.UpdateAndExecute1[T0,TRet](CallSite site, T0 arg0)
   at l99.driver.fanuc.strategies.FanucExtendedStrategy.peel(String veneerKey, Object input, Object[] additionalInputs) in /src/fanuc/strategies/FanucExtendedStrategy.cs:line 183
   at CallSite.Target(Closure , CallSite , Object )
   at l99.driver.fanuc.strategies.FanucExtendedStrategy.Peel(String veneerKey, Object[] inputs) in /src/fanuc/strategies/FanucExtendedStrategy.cs:line 204
   at CallSite.Target(Closure , CallSite , Object )
   at System.Dynamic.UpdateDelegates.UpdateAndExecuteVoid1[T0](CallSite site, T0 arg0)
   at l99.driver.fanuc.collectors.SpindleData.CollectForEachSpindleAsync(Int16 current_path, Int16 current_spindle, String spindle_name, Object spindle_split, Object spindle_marker) in /src/fanuc/collectors/SpindleData.cs:line 130
   at l99.driver.fanuc.strategies.FanucMultiStrategy.CollectForEachSpindleAsync(Int16 current_path, Int16 current_spindle, String spindle_name, Object spindle_split, Object spindle_marker) in /src/fanuc/strategies/FanucMultiStrategy.cs:line 107
   at System.Dynamic.UpdateDelegates.UpdateAndExecuteVoid1[T0](CallSite site, T0 arg0)
   at l99.driver.fanuc.strategies.FanucExtendedStrategy.CollectAsync() in /src/fanuc/strategies/FanucExtendedStrategy.cs:line 599|l99.driver.fanuc.strategies.FanucMultiStrategy|
```

**Comment by pts211** [comment]
Attaching trace logging of fanuc-driver startup with l99.driver.fanuc.collectors.AxisData enabled.
[driver.log](https://github.com/Ladder99/fanuc-driver/files/8823708/driver.log)

**Comment by MRIIOT** [comment]
Resolved in 0.5
